src/seedu/addressbook/ui/TextUi.java

```diff
@@ -76,46 +64,34 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        formatter.printWithPrefix("Enter command: ");
```

Comment:
you should be invoking the `Formatter` here to provide you the formatted string. Displaying the string is still the UI's responsibility.